src/app/public/modules/autocomplete/autocomplete-input.directive.ts

```diff
@@ -179,6 +184,12 @@ export class SkyAutocompleteInputDirective implements OnInit, OnDestroy, Control
       });
   }
 
+  public ngAfterViewInit(): void {
+    const autofillDirective = new SkyAutofillDirective(this.elementRef, this.renderer);
```

Comment:
This is an unusual implementation, and I'd be wary of it since the directive won't participate in Angular's directive lifecycle (`ngOnInit`, `ngAfterContentInit`, `ngOnDestroy`, etc.). Can we not leave it up to the consumer of the autocomplete directive to also add the autofill directive? If not, then we should probably move the logic for disabling autofill to a service and have both this directive and the autofill directive use it to disable autofill.